proto/zitadel/management.proto

```diff
@@ -6344,12 +6368,32 @@ message UpdateAppRequest {
     string project_id = 1 [(validate.rules).string = {min_len: 1, max_len: 200}];
     string app_id = 2 [(validate.rules).string = {min_len: 1, max_len: 200}];
     string name = 5 [(validate.rules).string = {min_len: 1, max_len: 200}];
+    string external_url = 6 [(validate.rules).string = {min_len: 1, max_len: 200}];
+    bool is_visible_to_end_user = 7;
```

Comment:
The field names are a bit confusing in my opinion, it's unclear that the fields are meant for an icon. Maybe it would make sense to create a sub message called icon for this case and add a `light_icon`-field and a `dark-icon`-field. I would also add it to the create app command.